### PR TITLE
Trigger procedure

### LINKED ISSUE #10: Trigger Procedure
Given a selected procedure, we want to be able to run that procedure on a clone of the original database

- Look for JOINs, and INSERTS to figure out which tables we need to clone.
- Create a list of words that we need to look for.

### PR DESCRIPTION
**Pull request by adityavidyadharan**
# Description

Slight modification to logging formatting.
Adds ability to trigger a procedure (with parameters) and fetch the parameter list for a procedure

## Link to the issue you are closing

Fixes #10 

## Screenshot or other testing that you have completed

Screen Shot 2022-09-25 at 6 39 51 AM

# Checklist:

- [x] ESLint passes
- [x] Prettier passes
- [x] Included comments where necessary
- [x] Updated README if needed
- [x] Verified that code runs and generates no errors/warnings

### CODE REVIEW

File: src/db/Procedures.ts [hunk 1]
```diff
@@ -90,6 +108,52 @@ export default class Procedures {
...
+
+  public async getProcedureParameters(
+    database: string,
+    procedure: string
+  ): Promise<ProcedureParameter[]> {
+    const client = await new Client({
+      host: this.address,
+      port: this.port,
+      password: 'asdf',
+      user: 'kpmg',
```
Comment (alex-hobby) [inline src/db/Procedures.ts:139]:
```suggestion
      password: this.password,
      user: this.user,
```
Can we do something like this? Would like to make it easier to pull these hard-coded values out later. Could make this change in all relevant functions here.

File: src/db/Procedures.ts [hunk 2]
```diff
@@ -11,6 +11,24 @@ type DBQuery = {
...
+  INOUT = 'INOUT',
+}
+type ProcedureParameter = {
+  direction: Direction;
+  name: string;
+  type: string;
+};
+
 export default class Procedures {
   address: string;
```
Comment (alex-hobby) [inline src/db/Procedures.ts:34]:
```suggestion
password: string = "asdf";
user: string = "kpmg";
```
This goes with the later comment, the point is that we can just delete the hardcoded values in one place.

**Review by alex-hobby — CHANGES_REQUESTED** [review]

File: src/db/Procedures.ts (hunk 1 above)
Comment (JackFratto) [reply src/db/Procedures.ts:139]: These problems are resolved in #73

Comment (adityavidyadharan) [reply src/db/Procedures.ts:139]: Once #73 is finalized and merged into `feature`, I'll pull it into my branch and update this code to match the other functions.

**Review by alex-hobby — APPROVED** [review]
👍

**Pull request merged**